Assets/Dev/B/Script/SimpleButton.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class SimpleButton : MonoBehaviour
+{
+    public GameObject button;
+
+    public void ControlMenu(GameObject _target)
```

Comment:
Let's avoid the preceding underscore when we can. In another class there was already a variable named character so it was ok to use it in order to distinguish the local variable. Here I don't think it's needed